flink-streaming-word-count/README.md

```diff
@@ -0,0 +1,48 @@
+# Example Streaming Application: (Flink Streaming API) Wordcount from Socket DataSource
+
+## Overview
+
+The example streaming application shows an example of an application that can be deployed using the PNDA Deployment Manager. (See the `platform-deployment-manager` project for details.)
+
+This example uses the Flink Streaming APIs. When PNDA is configured to use HDP/CDH cluster, Flink is deployed by default.
+
+The application is a tar file containing binaries and configuration files required to perform some stream processing.
+
+This example application reads events from a network socket and performs basic counting analytics. To run the data source run the command `nc -l 9100` on the pnda data node and then type some input into it.
+
+The results are printed into the console output of the flink driver process. To view these, navigate to the log file via the Yarn Resource Manager UI or use the PNDA log server.
+
+
+## Requirements
+
+* [Maven](https://maven.apache.org/docs/3.0.5/release-notes.html) 3.0.5
+* [Java JDK](https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html) 1.8
+
+## Build
+Edit the `streaming-app/pom.xml` file with the correct dependencies.
```

Comment:
Could you be more specific here? Which dependencies need setting and how does one know what 'correct' is?

Reply:
Changes are done.